src/events/ldap.js

```diff
@@ -0,0 +1,85 @@
+const uuid4 = require('uuid4');
+const serverlessEvent = require('../proto/event_pb.js');
+const utils = require('../utils.js');
+const tracer = require('../tracer.js');
+const errorCode = require('../proto/error_code_pb.js');
+const eventInterface = require('../event.js');
+const moduleUtils = require('./module_utils.js');
+
+/**
+ * Wraps the ldap.js bind command function with tracing
+ * @param {Function} bindFunction The wrapped bind function from ldap.js module
+ * @returns {Function} The wrapped function
+ */
+function bindWrapper(bindFunction) {
+    return function internalBindWrapper(dn, password, controls, callback) {
+        try {
+            utils.debugLog(`LDAP.js bind() wrapper - dn: ${dn}`);
+
+            const resource = new serverlessEvent.Resource([
+                this.url,
```

Comment:
consider using only the domain instead of the full URL as the resource name

Reply:
(or if that's what the url already is that's fine)

Reply:
it is only the domain